### PR TITLE
Cmake: Fixed CMake warning messages

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to fix the warning messages of CMake while running
'$ debuild -uc -us' command.

* Issue:
```bash
$ debuild -uc -us [enter]
  ... Omission ...
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_INSTALL_LOCALSTATEDIR
    CMAKE_INSTALL_SYSCONFDIR
```

**Changes proposed in this PR:**
1. Fixed a CMake warning while running debuild command

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---
# [Template] PR Description

In general, github system automatically copies your commit message for your convenience.
Please remove unused part of the template after writing your own PR description with this template.
```bash
$ git commit -s filename1 filename2 ... [enter]

Summarize changes in around 50 characters or less

More detailed explanatory text, if necessary. Wrap it to about 72
characters or so. In some contexts, the first line is treated as the
subject of the commit and the rest of the text as the body. The
blank line separating the summary from the body is critical;
various tools like `log`, `shortlog` and `rebase` can get confused 
if you run the two together.

Further paragraphs come after blank lines.

**Changes proposed in this PR:**
- Bullet points are okay, too
- Typically a hyphen or asterisk is used for the bullet, preceded
  by a single space, with blank lines in between, but conventions vary here.

Resolves: #123
See also: #456, #789

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

**How to evaluate:**
1. Describe how to evaluate in order to be reproduced by reviewer(s).

Add signed-off message automatically by running **$git commit -s ...** command.

$ git push origin <your_branch_name>
```

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,14 @@ ELSE(TAOS_CI_INSTALL_DIR)
     SET(TAOS_CI_INSTALL_DIR "/var/www/html/TAOS-CI")
 ENDIF(TAOS_CI_INSTALL_DIR)
 
+IF (NOT CMAKE_INSTALL_SYSCONFDIR)
+  set(CMAKE_INSTALL_SYSCONFDIR "etc" CACHE PATH "Read-only single-machine data (etc)")
+ENDIF()
```
Comment (ohsewon) [inline CMakeLists.txt]: ENDIF(NOT CMAKE_INSTALL_SYSCONFDIR) is looks better.

File: CMakeLists.txt [hunk 2]
```diff
@@ -7,6 +7,14 @@ ELSE(TAOS_CI_INSTALL_DIR)
...
+
+IF (NOT CMAKE_INSTALL_LOCALSTATEDIR)
+  set(CMAKE_INSTALL_LOCALSTATEDIR "var" CACHE PATH "Modifiable single-machine data (var)")
+ENDIF()
```
Comment (ohsewon) [inline CMakeLists.txt]: ENDIF(NOT CMAKE_INSTALL_LOCALSTATEDIR) is looks better.

Comment (leemgs) [reply CMakeLists.txt]: updated.

File: CMakeLists.txt (hunk 1 above)
Comment (leemgs) [reply CMakeLists.txt]: updated.

File: CMakeLists.txt [hunk 3]
```diff
@@ -7,6 +7,14 @@ ELSE(TAOS_CI_INSTALL_DIR)
     SET(TAOS_CI_INSTALL_DIR "/var/www/html/TAOS-CI")
 ENDIF(TAOS_CI_INSTALL_DIR)
 
+IF (NOT CMAKE_INSTALL_SYSCONFDIR)
```
Comment (ohsewon) [inline CMakeLists.txt]: need to keep indentation. exsting 'if' statement doesn't have blank. 
ex) IF(NOT CMAKE_INSTALL_SYSCONFDIR)

File: CMakeLists.txt [hunk 4]
```diff
@@ -7,6 +7,14 @@ ELSE(TAOS_CI_INSTALL_DIR)
...
+  set(CMAKE_INSTALL_SYSCONFDIR "etc" CACHE PATH "Read-only single-machine data (etc)")
+ENDIF(CMAKE_INSTALL_SYSCONFDIR)
+
+IF (NOT CMAKE_INSTALL_LOCALSTATEDIR)
```
Comment (ohsewon) [inline CMakeLists.txt]: same as above.

File: CMakeLists.txt (hunk 3 above)
Comment (leemgs) [reply CMakeLists.txt]: Nice catch.

File: CMakeLists.txt (hunk 4 above)
Comment (leemgs) [reply CMakeLists.txt]: Nice catch. update. resubmitted.

**Pull request merged**